crates/bevy_input/src/gamepad.rs

```diff
@@ -1,12 +1,21 @@
 use crate::{Axis, Input};
 use bevy_app::{EventReader, EventWriter};
 use bevy_ecs::system::{Res, ResMut};
-use bevy_utils::HashMap;
+use bevy_utils::{tracing::info, HashMap, HashSet};
 
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
 pub struct Gamepad(pub usize);
 
+#[derive(Default)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+/// Container of unique connected [Gamepad]s
+///
+/// [Gamepad]s are registered and deregistered in [gamepad_connection_system]
+pub struct GamepadLobby {
+    pub gamepads: HashSet<Gamepad>,
+}
```

Comment:
I don't think this needs to be Serializable.

Reply:
Removed serialize